.changeset/quiet-hairs-wave.md

```diff
@@ -0,0 +1,5 @@
+---
+'vercel': major
```

Comment:
This PR moves parsing into the individual subcommands, so I think the major is justified.

Reply:
I forgot to fill out the PR description. So it depends on how pedantic we want to be here. I put major here because technically this would "work" previously:

```
vc alias --next 12345 mydeployment-ns0sbhzj6-trek.vercel.app alias-test.vercel.app
```

It's not correct usage, and the `--next` flag would be ignored, but the command would succeed.

With this change that same command now fails (which is correct, but could be considered a breaking change technically):

```
Error: unknown or unexpected option: --next
```